R/looping.R

```diff
@@ -1,10 +1,10 @@
-#' generate forecaster predictions on a particular dataset
-#' @description
-#' a wrapper that turns a forecaster, parameters, data
-#' combination into an actual experiment that outputs a
-#' prediction.
-#' as far as batchtools is concerned, the scoring function is a particular
-#'   parameter of the forecaster (or Algorithm, as they call it).
+#' Generate forecaster predictions on a particular dataset
+#'
+#' A wrapper that turns a forecaster, parameters, data combination into an
+#' actual experiment that outputs a prediction. as far as batchtools is
+#' concerned, the scoring function is a particular parameter of the forecaster
+#' (or Algorithm, as they call it).
+#'
```

Comment:
```suggestion
#' A wrapper that turns a forecaster, parameters, data combination into an
#' actual experiment that outputs a prediction for each day.
#'
```
old batchtools refence is old